Answers.md

```diff
@@ -0,0 +1,10 @@
+1. Semantic HTML is a way to make the anchors and tags of your website more "descriptive" so it's meta-data can be more easily found and comprehended. Instead of just <div> everywhere, semantic tags like <header> <section> <article> <footer> <figure> more accurately describe the "meaning" of your content, not just what goes where. 
+
+2. The main difference is, an element with {display: block} takes up the entire width of the browser - or as much space as possible; and {display: inline} only takes up the space of the content (i.e., text or image) contained in that element. 
+
```

Comment:
Yup spot on. But be careful with the `display: block`. I wouldn't just say it takes the entire width of the browser. It more takes the width of the entire container.